### PR TITLE
Event Report for April,2017 Python Pune Meetup

### PR DESCRIPTION
**Pull request by rahulbajaj0509**

### CODE REVIEW

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+layout: entry
+title: March Python Pune Meetup 2016
+category: meeting-notes
+---
+
+The [April PythonPune Meetup](https://www.meetup.com/PythonPune/events/239079298/) was held on 30th April at [Red Hat, Pune](http://www.redhat.com/en) .
+The meetup was scheduled at 10:30am and by the time it was 11am it was house full!,
+ about 70 attendees were present at the meetup.
+
+### The Agenga of the meetup was:
```
Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/Agenga/Agenda/

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+layout: entry
+title: March Python Pune Meetup 2016
+category: meeting-notes
+---
+
+The [April PythonPune Meetup](https://www.meetup.com/PythonPune/events/239079298/) was held on 30th April at [Red Hat, Pune](http://www.redhat.com/en) .
```
Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/Pune ./Pune./

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+The meetup was scheduled at 10:30am and by the time it was 11am it was house full!,
```
Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/10:30am/10:30 am/

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+ about 70 attendees were present at the meetup.
+
+### The Agenga of the meetup was:
+
+ * All about Virtualenv - by [Poruri Sai Rahul](https://github.com/rahulporuri)
```
Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/Virtualenv/virtualenv/

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+ * Deep Learning using Keras - by [Prakash Anand](https://www.meetup.com/PythonPune/members/222517572/)
+ * Game Theory and Iterated Prisoner's Dilemma - by [Sourav Sing](https://twitter.com/MrSouravSingh)
+ * Introduction to Ansible - by [Abhijeet Kasurde](https://github.com/Akasurde)
+
+### The first talk of the meetup was presented by [Poruri Sai Rahul](https://github.com/rahulporuri), he spoke about:
+
+ - What is virtual environment.
```
Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/virtual/a virtual/

Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/environment./environment?/

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+ - How to set up a virtual environment using python Virtualenv.
```
Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/python/Python/

Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/Virtualenv/virtualenv?

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+ - What is the importance of Virtualenv.
```
Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/Virtualenv/virtualenv?

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+ - How important is the placement of the Virtualenv folder.
```
Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: Again missing ? here and virtualenv in small

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+The talk was an **interactive one**, many questions were asked like :
+
+ - What is the diffence between a stand alone program and a virtul environment ?
```
Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/diffence/difference

Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/stand alone/stand-alone/

Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/virtul/virtual/

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+ - Is docker a substitute to virtual environments ?
```
Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/docker/Docker/
s/environments ?/environments?

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+
+The slides to [Poruri Sai Rahul](https://github.com/rahulporuri) talk are present [here](https://github.com/rahulporuri/talks/blob/master/virtualenvs_runtimes.pdf).
+
+### The second talk of the meetup was presented by [Prakash Anand](https://www.meetup.com/PythonPune/members/222517572/), he spoke about:
+
+The third talk of the meetup was presented by [Sourav Sing](https://twitter.com/MrSouravSingh), he spoke about:
+
+ - What is game theory.
```
Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: Should end with ?

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+ - What is Prisoner Dilemma.
```
Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: Same here

Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: Also, Prisoner's dilemma

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+ - What is payoff matrix.
```
Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: Ends with ?

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+ - Also, he performed a demo-session in which many graphs were simulated using IPD in python.
```
Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/demo-session/demo session/

Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/python/Python/

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 15]
```diff
@@ -0,0 +1,51 @@
...
+
+The slides to [Sourav Sing's](https://twitter.com/MrSouravSingh) talk are present [here](https://gitpitch.com/souravsingh/IPD-presentation/master).
+
+### The last talk of the meetup was presented by [Abhijeet Kasurde](https://github.com/Akasurde), he spoke about:
+
+ - Introduction to DevOPS.
```
Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: Why OPS in caps?

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 16]
```diff
@@ -0,0 +1,51 @@
...
+ - Introduction to Ansile.
```
Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: What is Ansile?

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 17]
```diff
@@ -0,0 +1,51 @@
...
+ - Why is ansible better than other configuration management Systems.
```
Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/ansible/Ansible/

Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: Ends with ?

Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: nitpicking but probably `is better` would be good

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 18]
```diff
@@ -0,0 +1,51 @@
...
+ - He spoke extensively about provisioning, configuration management and application continuous delivery.
```
Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/management/management,/

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 19]
```diff
@@ -0,0 +1,51 @@
...
+
+The meetup was very informative and interactive. Not to forget we were served with tasty food also.
+I would like to thank the organizers of the meetup for arranging such events where we always get to learn somehting new.
```
Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/somehting/something/

File: _posts/2017-04-30-april-pythonpune-meetup.md (hunk 17 above)
Comment (sayanchowdhury) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: Also `-` the first `is`

**Comment by rahulbajaj0509** [comment]
@sayanchowdhury  i have made the changes, thank you for the review :)

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 20]
```diff
@@ -0,0 +1,51 @@
+---
+layout: entry
+title: March Python Pune Meetup 2016
```
Comment (shakthimaan) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/2016/2017/ ?

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 21]
```diff
@@ -0,0 +1,51 @@
...
+category: meeting-notes
+---
+
+The [April PythonPune Meetup](https://www.meetup.com/PythonPune/events/239079298/) was held on 30th April at [Red Hat, Pune](http://www.redhat.com/en).
+The meetup was scheduled at 10:30 am and by the time it was 11 am it was house full!,
```
Comment (shakthimaan) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/full!/full, and/

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 22]
```diff
@@ -0,0 +1,51 @@
...
+
+ * All about virtualenv - by [Poruri Sai Rahul](https://github.com/rahulporuri)
+ * Deep Learning using Keras - by [Prakash Anand](https://www.meetup.com/PythonPune/members/222517572/)
+ * Game Theory and Iterated Prisoner's Dilemma - by [Sourav Sing](https://twitter.com/MrSouravSingh)
+ * Introduction to Ansible - by [Abhijeet Kasurde](https://github.com/Akasurde)
+
+### The first talk of the meetup was presented by [Poruri Sai Rahul](https://github.com/rahulporuri), he spoke about:
+
+ - What is a virtual environment?
+ - How to set up a virtual environment using python virtualenv?
```
Comment (shakthimaan) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/python/Python/

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 23]
```diff
@@ -0,0 +1,51 @@
...
+ - What is the importance of virtualenv?
+ - How important is the placement of the virtualenv folder?
+
+The talk was an **interactive one**, many questions were asked like :
```
Comment (shakthimaan) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: ", ... and many questions were asked such as"

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 24]
```diff
@@ -0,0 +1,51 @@
...
+
+ - What is the difference between a stand-alone program and a virtual environment?
+ - Is Docker a substitute to virtual environments?
```
Comment (shakthimaan) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/to/for/

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 25]
```diff
@@ -0,0 +1,51 @@
...
+
+The slides to [Poruri Sai Rahul](https://github.com/rahulporuri) talk are present [here](https://github.com/rahulporuri/talks/blob/master/virtualenvs_runtimes.pdf).
+
+### The second talk of the meetup was presented by [Prakash Anand](https://www.meetup.com/PythonPune/members/222517572/), he spoke about:
```
Comment (shakthimaan) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/, he/and he/

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 26]
```diff
@@ -0,0 +1,51 @@
...
+
+The third talk of the meetup was presented by [Sourav Sing](https://twitter.com/MrSouravSingh), he spoke about:
```
Comment (shakthimaan) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/, he/, and he/

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 27]
```diff
@@ -0,0 +1,51 @@
...
+
+ - What is game theory?
+ - What is Prisoner's Dilemma?
+ - What is payoff matrix?
+ - Also, he performed a demo session in which many graphs were simulated using IPD in Python.
```
Comment (shakthimaan) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/performed/gave/

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 28]
```diff
@@ -0,0 +1,51 @@
...
+
+The slides to [Sourav Sing's](https://twitter.com/MrSouravSingh) talk are present [here](https://gitpitch.com/souravsingh/IPD-presentation/master).
+
+### The last talk of the meetup was presented by [Abhijeet Kasurde](https://github.com/Akasurde), he spoke about:
```
Comment (shakthimaan) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/, he/, and he/

File: _posts/2017-04-30-april-pythonpune-meetup.md [hunk 29]
```diff
@@ -0,0 +1,51 @@
...
+
+ - Introduction to Devops.
+ - Introduction to Ansible.
+ - Why is ansible better than other configuration management Systems?
```
Comment (shakthimaan) [inline _posts/2017-04-30-april-pythonpune-meetup.md]: s/ansible/Ansible/
s/Systems/tools/

**Comment by rahulbajaj0509** [comment]
thanks @shakthimaan @sayanchowdhury  for reviewing this pr. I have made the appropriate changes, let me know if anything else is required :)

**Pull request merged**